inscriptions/ISic000530.xml

```diff
@@ -191,11 +196,14 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><orig>L</orig><gap reason="illegible" quantity="1" unit="character"/>II<gap reason="illegible" quantity="1" unit="character"/><hi rend="reversed">D</hi>I<gap reason="illegible" quantity="1" unit="character"/>
+                    <lb n="0"/><gap reason="lost" quantity="1" unit="line"/>
```

Comment:
given CIL's practices, i think this should be line 2 and gap reason illegible, extent unknown, unit character